rdopkg_tar/tar_changes.py

```diff
@@ -123,6 +123,17 @@ def commit_distgit_amend(suffix):
     cmd = ['commit', '-a', '-F', '-', '--amend']
     git(*cmd, input=message, print_output=True)
 
+def upload_source(osdist,tarball,args_new_sources):
+    """ Avoid upload of new sources to dist-git lookaside cache if --no-new-sources is passed in command line
+    Else upload the new sources to dist-git cache 
+    :returns: fedpkg command or False to skip the upload """
+    if guess.new_sources() and args_new_sources:
```

Comment:
lets take the guess.new_sources() condition out and put it back in main to avoid needing to pass `guess` to the upload_source function

Reply:
guess.new_sources() is not being passed as a parameter to the method, since we have import guess.

Reply:
oh I see, I thought it was being initialized in main(), my bad, you can ignore that suggestion then :)